appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/service/AppSecKitInitializer.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * -
+ * Copyright (C) 2023 Vaadin Ltd
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full license.
+ */
+
+package com.vaadin.appsec.v7.service;
+
+import javax.servlet.annotation.WebListener;
+import javax.servlet.http.HttpSessionEvent;
+import javax.servlet.http.HttpSessionListener;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vaadin.appsec.backend.AppSecService;
+import com.vaadin.server.VaadinService;
+
+@WebListener
+public class AppSecKitInitializer implements HttpSessionListener {
```

Comment:
Could we add some Javadoc here as well, something similar what we have in v8 init lisener?
```suggestion
A Vaadin initializer for initializing AppSec Kit services. Will be invoked automatically by Vaadin.
```

Reply:
Done